app/services/save_practice_service.rb

```diff
@@ -133,4 +135,19 @@ def crop_main_display_image
       @practice.main_display_image.reprocess!
     end
   end
+
+  def update_initiating_facility
+    @initiating_facility_type = @practice_params[:initiating_facility_type]
+    @initiating_facility = @practice_params[:initiating_facility]
+    if @practice_params[:initiating_facility_type] != 'department'
```

Comment:
nit: we could just do `initiating_facility_type` instead of `@practice_params[:initiating_facility_type]` here since we define it above